whylog/tests/tests_log_reader/__init__.py

```diff
@@ -104,12 +126,66 @@ def test_one(self, test_name):
             )
         )
 
-        # action and checking the result
         results = log_reader.get_causes(effect_line)
         expected_results = self._investigation_results_from_yaml(results_yaml_file, result_log_file)
-        assert results
-        assert len(results) == len(expected_results)
-        for got, real in six.moves.zip(results, expected_results):
-            assert got.lines == real.lines
-            for constr_got, constr_real in zip(got.constraints, real.constraints):
-                assert constr_got['name'] == constr_real['name']
+        self._check_results(results, expected_results)
+
+    @generate(
+        '001_most_basic',
+        # '002_match_latest',
+        '003_match_time_range',
+        # '005_match_tree',
+        '006_match_parameter',
+        '007_match_or',
+        '008_match_and',
+        '009_match_negation',
+        '010_multiple_files',
+        '011_different_entry',
+        # '012_multiple_rulebooks',
+        # '013_match_and_incomplete',
+    )  # yapf: disable
```

Comment:
This code is repeated. If we want to run the same tests here and above, please move tests names into separated tuple and use it here and above.